indico/modules/events/abstracts/views.py

```diff
@@ -28,11 +28,13 @@ class WPManageAbstracts(WPJinjaMixin, WPConferenceModifBase):
     def getJSFiles(self):
         return (WPConferenceModifBase.getJSFiles(self) +
                 self._asset_env['markdown_js'].urls() +
+                self._asset_env['selectize_js'].urls() +
```

Comment:
we do that for some other fields, i just kept it as it was done before for this one